src/http/sr-modules-operators-keys/entities/sr-module-operators-keys.response.ts

```diff
@@ -37,3 +37,32 @@ export class SRModuleOperatorsKeysResponse {
   })
   meta!: ELMeta;
 }
+
+export class SRModulesOperatorsKeysStreamResponse {
+  @ApiProperty({
+    type: 'array',
+    items: { oneOf: [{ $ref: getSchemaPath(Operator) }] },
+    description: 'Operators of staking router module',
+  })
+  operators?: Operator[];
+
+  @ApiProperty({
+    type: 'array',
+    items: { oneOf: [{ $ref: getSchemaPath(Key) }] },
```

Comment:
I think  type: () => [Key] should be enough

Reply:
done